actions/semantic_versioning/action.yml

```diff
@@ -24,11 +27,15 @@ runs:
       id: semantic
       shell: bash
       run: |
+        #!/bin/bash
```

Comment:
do we need that if we have `shell: bash` two lines above?

Reply:
I think yes. It help us test script locally as is